genesyscloud/architect_datatable/resource_genesyscloud_architect_datatable_test.go

```diff
@@ -104,6 +108,29 @@ func TestAccResourceArchitectDatatable(t *testing.T) {
 	})
 }
 
+func testVerifyDatatablesDestroyed(state *terraform.State) error {
+	archAPI := platformclientv2.NewArchitectApi()
+	for _, rs := range state.RootModule().Resources {
+		if rs.Type != "genesyscloud_architect_datatable" {
+			continue
+		}
+
+		//TODO FIX THIS !!!!
```

Comment:
is this TODO something that should be done before merge, or when the SDK can be used ?